src/families/hedera/api/network.ts

```diff
@@ -0,0 +1,54 @@
+import BigNumber from "bignumber.js";
+import * as hedera from "@hashgraph/sdk";
+import { Account } from "../../../types";
+import { Transaction } from "../types";
+import { calculateAmount } from "../utils";
+import { AccountId } from "@hashgraph/sdk";
+
+export function broadcastTransaction(
+  transaction: hedera.Transaction
+): Promise<hedera.TransactionResponse> {
+  return transaction.execute(getClient());
+}
+
+export function buildUnsignedTransaction({
+  account,
+  transaction,
+}: {
+  account: Account;
+  transaction: Transaction;
+}): hedera.TransferTransaction {
+  const { amount } = calculateAmount({ account, transaction });
+  const hbarAmount = hedera.Hbar.fromTinybars(amount);
+  const accountId = account.hederaResources!.accountId;
+
+  return new hedera.TransferTransaction()
+    .setTransactionId(hedera.TransactionId.generate(accountId))
+    .addHbarTransfer(accountId, hbarAmount.negated())
+    .addHbarTransfer(transaction.recipient, hbarAmount)
+    .freezeWith(getClient());
+}
+
+export interface AccountBalance {
+  balance: BigNumber;
+}
+
+export async function getAccountBalance(
+  accountId: AccountId
+): Promise<AccountBalance> {
+  const accountBalance = await new hedera.AccountBalanceQuery({
+    accountId,
+  }).execute(getClient());
+
+  return {
+    balance: accountBalance.hbars.to(hedera.HbarUnit.Tinybar),
+  };
+}
+
+let _hederaClient: hedera.Client | null = null;
+
+function getClient(): hedera.Client {
+  _hederaClient ??= hedera.Client.forMainnet().setMaxNodesPerTransaction(1);
```

Comment:
The client may be targeting a range from node `0.0.3` to `0.0.20` and I don't believe there is a way to provide a custom URL.

I may need an expert opinion to confirm my assumptions though.

Reply:
Topic still pending, we should discuss it during our sync meeting. Either a reverse proxy isn't considered mandatory on this part, or you could maybe target the network directly, same way as it's done in `mirror.ts`.

Reply:
Apparently we _can_ set a proxy via `client.setNetwork()`, but I wanted to ask whether the proxy has been set up? If so, can it be provided so I can try it out to see if it can work please

Reply:
Cool, yes we now have a proxy up and running : `https://hedera.coin.ledger.com`